index.js

```diff
@@ -44,8 +44,7 @@ queue.process('email', 30, require('./workers/email'));
 queue.process('sendBatchEmails', require('./workers/sendBatchEmails')(queue, db, baseUrl));
 queue.process('sendWeeklyRecapEmails', require('./workers/sendWeeklyRecapEmails')(queue, db, baseUrl));
 queue.process('amazonPayments', require('./workers/amazonPayments')(queue, db));
-
-queue.promote();
```

Comment:
Consider that we're using Kue 0.8, not 0.10, so it's likely it was deprecated later